src/main/modules/oidc/index.ts

```diff
@@ -24,7 +24,7 @@ export class OidcMiddleware {
     const redirectUri: string = config.get('services.idam.callbackURL');
 
     server.get('/login', (req, res) => {
-      res.redirect(loginUrl + '?client_id=' + clientId + '&response_type=code&redirect_uri=' + encodeURI(redirectUri));
+      res.redirect(loginUrl + '?client_id=' + clientId + '&response_type=code&redirect_uri=' + encodeURI(redirectUri) + '&scope=openid%20roles%20profile%20search-user%20manage-user');
```

Comment:
Just double checking, we needed this didn't we? I think! I remember we were debugging

Reply:
Seems to work without the scopes so I have removed them

Reply:
Actually the scope are necessary so I have added them back in